### DISCUSSION TITLE
Performance comparison with the normal uBO?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBOL-home#110] [General]
I can feel uBOL is slower than the normal uBO but the different is miniscule in Chrome. It will be more noticeable when you spawn a large number of tabs the same time, though. This is about Chrome in PC. On Android, the only Chromium based browser I'm using is Kiwi Browser, and I can only use Basic mode with it so it doesn't count. But, on Firefox Android, the performance different between uBOL and normal uBO is very noticeable. I can see empty frames and banner ads before they are hidden.

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#is-ubol-more-efficient-cpu--and-memory-wise-than-ubo

**Reply by garry-ut99** [reply]
Maybe it's what causes this problem: 
- https://github.com/uBlockOrigin/uBOL-home/discussions/226
